[OAP-cache] Modify visibility of FiberCacheManager from private sql to public

This is a blocker issue for `mvn clean install`, the error msg as following:
```
/oap-mine/oap-cache/oap/src/main/java/org/apache/parquet/hadoop/ParquetCacheableFileReader.java:54: error: not found: type FiberCacheManager
  private FiberCacheManager cacheManager = OapRuntime$.MODULE$.getOrCreate().fiberCacheManager();
```
For quick fix, Modify visibility of FiberCacheManager from private sql to public

--- a/oap-cache/oap/src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/FiberCacheManager.scala
+++ b/oap-cache/oap/src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/FiberCacheManager.scala
@@ -35,7 +35,7 @@ import org.apache.spark.unsafe.Platform
 import org.apache.spark.util.Utils
 import org.apache.spark.util.collection.OapBitSet
 
-private[sql] class FiberCacheManager(
+class FiberCacheManager(
     sparkEnv: SparkEnv) extends Logging {
   private val GUAVA_CACHE = "guava"
   private val SIMPLE_CACHE = "simple"

